phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -899,6 +906,7 @@ private void rebuildIndexRows(Map<byte[], List<Mutation>> indexKeyToMutationMap,
                 }
             }
             if (batchSize > 0) {
+                ungroupedAggregateRegionObserver.checkForRegionClosing();
```

Comment:
You missed the for loop before this (line 903)

Reply:
good catch! Thanks!